.github/workflows/packaging-wheels.yml

```
@@ -35,12 +35,32 @@ jobs:
+      # sccache replaces it on Windows, deliberately without cross-run
+      # persistence. Measured: 93 of 573 hits cold, 124 warm -- so persisting it
+      # buys 31 hits for a ~500 MB entry, in a repository already at its 10 GB
+      # cache limit where that pressure starves the C++ jobs. Nearly all of the
+      # benefit is dedup within the job, between the four wheel builds, and that
+      # costs nothing. The Python-dependent compilations cannot cache at all:
+      # uv's build-isolation directory is named randomly per run, so their
+      # include path differs on every build.
+      - if: runner.os == 'Windows'
+        name: Configure sccache
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)